app/presenters/brexit_taxon_presenter.rb

```diff
@@ -17,4 +18,12 @@ def description
   def finder_link
     "https://finder-frontend-pr-706.herokuapp.com/prepare-individual-uk-leaving-eu?topic=#{taxon.base_path}"
   end
+
+  def data_attributes
+    {
+      "track-category": "navGridContentClicked",
+      "track-action": @index,
```

Comment:
Do we usually set a number as "track-action"? I wonder if instead of passing an index we could pass the id of the taxon or something like that, to simplify this class.

Reply:
The track-action is the link's position within other similar links. E.g: if there are 6 taxons, when you click on the first link it should have a track-action of 1, whereas if you click on the last taxon link, it should have a track-action of 6 (which is also why I've offset the index by 1).